### PR TITLE
Add alpha data-vis colors

### PR DESCRIPTION
**Pull request by langermank**
Adds the first iteration of data-vis colors for internal testing

### CODE REVIEW

**Comment by lukasoppermann** [comment]
Hey @langermank,

there were actually a few things that needed fixing:
- figma needs to have a different setup for shadows, this is why it does not use the theme file everywere, so we need to add the display as well, as we have shadows in pattern: [scripts/buildPlatforms/buildFigma.ts](https://github.com/primer/primitives/pull/820/commits/0e12b1809171c8444da16b75f817e26839f703ca#diff-7c33bda857c2426024675ae7cb2bcb60ce2104dacfd61b6c071d43c6c848b4cb)
- deprecated files needed the reference as well: https://github.com/primer/primitives/pull/820/commits/0e12b1809171c8444da16b75f817e26839f703ca#diff-711832e96e66a85c309d33133a318d8c9b995f6e26ff38c87ddcf81a32b9cd1cR165
- you had some wrong references in the themes file: src/tokens/base/color/**light**/display-**dark**.json5 ([link](https://github.com/primer/primitives/pull/820/commits/0e12b1809171c8444da16b75f817e26839f703ca#diff-89c9c423e68773b16c86dc50ebad45c47bdc39882b64917c00d35d6a02cc66c4))

Once we remove the old tokens and everything is more clean, we can look into improving this so that we only define those themes once.

File: docs/storybook/stories/Color/DataVis/Charts.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  'highcharts-neutral-color-5',
+  'highcharts-neutral-color-3',
+  'highcharts-highlight-color-100',
+  'highcharts-highlight-color-80',
+  'highcharts-highlight-color-60',
+  'bgColor-accent-muted',
+  'bgColor-accent-muted',
+  'bgColor-success-emphasis',
+  'bgColor-danger-emphasis',
+]
```
Comment (langermank) [inline docs/storybook/stories/Color/DataVis/Charts.stories.tsx:77]:
```suggestion
```

File: docs/storybook/stories/Color/DataVis/Charts.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+// eslint-disable-next-line import/extensions
+import colorTokens from '../../../../../tokens-next-private/docs/functional/themes/light.json'
+import {ColorTokenSwatch} from '../../StorybookComponents/ColorTokenSwatch/ColorTokenSwatch'
+import {DataTable, Table} from '@primer/react/drafts'
+import {InlineCode} from '../../StorybookComponents/InlineCode/InlineCode'
+import {getTokensByName} from '../../utilities/getTokensByName'
+
+export default {
+  title: 'Color/DataVis',
```
Comment (langermank) [inline docs/storybook/stories/Color/DataVis/Charts.stories.tsx:10]:
```suggestion
  title: 'Color/DataVis',
  tags: ['excludeSnapshot']
```

File: docs/storybook/stories/Color/DataVis/Charts.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  tags: ['excludeSnapshot']
```
Comment (langermank) [inline docs/storybook/stories/Color/DataVis/Charts.stories.tsx:11]:
```suggestion
  tags: ['excludeSnapshot'],
```

**Pull request merged**